rise/lib/resolvers/categories.ts

```diff
@@ -1,21 +1,40 @@
 import { gql } from 'apollo-server-express'
-import { getCategories } from '../services/categories'
+import { getCategories, getCategoryById } from '../services/categories'
+import { getServicesByCategory } from '../services/riseServices'
 
 export const typeDefs = gql`
+  type Service {
+    id: String!
+    name: String!
+    link: String!
+    categories: [Category!]!
+  }
+
   type Category {
     id: String!
     label: String!
     introduction: String!
     information: String!
+    services: [Service]
   }
 
   extend type Query {
     categories: [Category!]!
+    categoryAndRelated(id: String!): Category!
```

Comment:
I think this is better described as `Category` unless it returns a default category if provided with an incorrect ID!

```suggestion
    categoryAndRelated(id: String!): Category
```

Reply:
Yeah thought about that aswel, the issue with that is that I thought we might add articles etc in the future. So felt dumb naming it servicesAndArticlesForCategory 🤪